Cargo.toml

```diff
@@ -459,11 +459,11 @@ solana_rbpf = "=0.8.5"
 spl-associated-token-account = "=4.0.0"
```

Comment:
Is there a new version of SPL-ATA? It seems like the monorepo is holding onto duplicates of a bunch of the bumped crates because of ATA `=v4.0.0`

I don't remember everything we've discovered about juggling these, but historically it looks like we've bumped ATA when we need to bump token-2022

Reply:
We didn't need to this time because we introduced a client crate with https://github.com/solana-labs/solana-program-library/pull/7005, which simplifies this whole tangle quite a bit! Buf if you want, I can bump the ATA program crate's major version and upgrade here too.

Side note: I eventually want to do a separate PR to switch the monorepo to using the ATA client crate instead of the ATA program crate, but the client crate doesn't have the instruction enum, to avoid a dependency on borsh. This means that we can't do an exhaustive match on the deserialized instruction in `transaction-status`. I'll have to figure out a decoding solution there first.

Reply:
I'm not even sure it needs to be a new major version, at least from the monorepo perspective.
But yeah, it seems like unnecessary bloat to me to import 2 different copies of each of 6 crates.

Reply:
Ok, done!